source/usage-examples/count.txt

```diff
@@ -23,6 +23,19 @@ than scanning the entire collection. The ``countDocuments()`` method returns
 an **accurate** count of the number of documents and supports specifying
 a filter.
 
+.. tip::
+
+   You can improve performance when using ``countDocuments()`` to return the total
+   number of documents in a collection by avoiding a collection scan. To do
```

Comment:
Suggestion: In my opinion, the structure of this sentence is a little confusing because the suggestion for how to improve performance is so far from the description. 

```suggestion
   When using ``countDocuments()`` to return the total
   number of documents in a collection, you can improve performance by avoiding a collection scan. To do
```